controllers/flinkcluster_reconciler.go

```diff
@@ -70,8 +70,8 @@ func (reconciler *ClusterReconciler) reconcile() (ctrl.Result, error) {
 		reconciler.log.Info("The cluster update is in progress")
 	}
 	// If batch-scheduling enabled
-	if reconciler.observed.cluster.Spec.BatchSchedulerName != "" {
-		scheduler, err := batchscheduler.GetScheduler(reconciler.observed.cluster.Spec.BatchSchedulerName)
+	if reconciler.observed.cluster.Spec.BatchSchedulerName != nil {
```

Comment:
What would happen if it is ""?

Reply:
With this, the comments is outdated, now "" means invalid scheduler name.

Reply:
addressed,